<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="matALabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Matrix A</string>
    </property>
   </widget>
   <widget class="QLabel" name="aEquLabel">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>125</y>
      <width>50</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>A =</string>
    </property>
   </widget>
   <widget class="QLabel" name="matALeftBracket">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>50</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>75</pointsize>
     </font>
    </property>
    <property name="text">
     <string>[</string>
    </property>
   </widget>
   <widget class="QLabel" name="matARightBracket">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>50</y>
      <width>50</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>75</pointsize>
     </font>
    </property>
    <property name="text">
     <string>]</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="matAEntryOne">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>100</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="matAEntryTwo">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>100</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="matAEntryThree">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>155</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QTextEdit" name="matAEntryFour">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>155</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="uEquLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>U =</string>
    </property>
   </widget>
   <widget class="QLabel" name="matULeftBracket">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>270</y>
      <width>50</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>75</pointsize>
     </font>
    </property>
    <property name="text">
     <string>[</string>
    </property>
   </widget>
   <widget class="QLabel" name="matURightBracket">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>270</y>
      <width>50</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>75</pointsize>
     </font>
    </property>
    <property name="text">
     <string>]</string>
    </property>
   </widget>
   <widget class="QLabel" name="sEquLabel">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>350</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>S =</string>
    </property>
   </widget>
   <widget class="QLabel" name="matSLeftBracket">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>270</y>
      <width>50</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>75</pointsize>
     </font>
    </property>
    <property name="text">
     <string>[</string>
    </property>
   </widget>
   <widget class="QLabel" name="matSRightBracket">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>270</y>
      <width>50</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>75</pointsize>
     </font>
    </property>
    <property name="text">
     <string>]</string>
    </property>
   </widget>
   <widget class="QLabel" name="vEquLabel">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>350</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>V^T =</string>
    </property>
   </widget>
   <widget class="QLabel" name="matVLeftBracket">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>270</y>
      <width>50</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>75</pointsize>
     </font>
    </property>
    <property name="text">
     <string>[</string>
    </property>
   </widget>
   <widget class="QLabel" name="matVRightBracket">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>270</y>
      <width>50</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>75</pointsize>
     </font>
    </property>
    <property name="text">
     <string>]</string>
    </property>
   </widget>
   <widget class="QLabel" name="matUEntryOne">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="matUEntryTwo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="matUEntryThree">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>375</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="matUEntryFour">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>375</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="matSEntryOne">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>320</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="matSEntryTwo">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>320</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="matSEntryThree">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>375</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="matSEntryFour">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>375</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="matVEntryOne">
    <property name="geometry">
     <rect>
      <x>605</x>
      <y>320</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="matVEntryTwo">
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>320</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="matVEntryThree">
    <property name="geometry">
     <rect>
      <x>605</x>
      <y>375</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="matVEntryFour">
    <property name="geometry">
     <rect>
      <x>665</x>
      <y>375</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="systemLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>750</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter values inside of Matrix A to generate a SVD.
Please note that this calculator currently only supports real unique eigenvalues ONLY.
The resulting decomposition is shown below.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>450</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QMenu" name="title">
    <property name="title">
     <string>SVD Calculator</string>
    </property>
   </widget>
   <addaction name="title"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
